examples/basics/scene/volume.py

```diff
@@ -66,6 +72,10 @@ def on_key_press(event):
     elif event.text == '3':
         volume1.visible = not volume1.visible
         volume2.visible = not volume1.visible
+    elif event.text == '4':
+        cmap = cmaps.next()
```

Comment:
I get an error that it has no attribute `next` -- this is a Python3 thing. You should use `from vispy.ext.six import next` and do `next(cmaps)`